deploy_and_run_integration_tests.sh

```diff
@@ -0,0 +1,95 @@
+#!/bin/bash
+
+# Copyright 2018 Google Inc.  All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# This script builds a Docker image from the current state of the local code,
+# pushes the image to a test registry (project 'test-gcp-variant-transforms')
+# and runs integration tests against that image.
+#
+# To run this test successfully:
+# - 'docker' should be installed.
+#
+# - The user that runs this script should be in the 'docker' group.
+#
+# - The user's gcloud credentials should be set; follow the steps at:
+#   https://cloud.google.com/genomics/install-genomics-tools
+#
+# - The user should have access to the 'test-gcp-variant-transforms' project.
+#
+# - If you get DNS name resolution errors, add these dns options to
+#   /etc/default/docker and restart docker daemons:
+#   DOCKER_OPTS="--dns 8.8.8.8 --dns 8.8.4.4"
+
+# Checks and inits
+PROJECT="test-gcp-variant-transforms"
+GS_DIR="integration_test_runs"
+GREEN="\e[32m"
+RED="\e[31m"
+NC="\e[0m"  # No Color, i.e., reset the color
+
+cur_dir=$(pwd)
+if [[ ! (${cur_dir} =~ "gcp-variant-transforms") ]]; then
+  echo "This script should be run from the gcp-variant-transforms root!"
+  exit
+fi
+docker_file=$(ls docker/Dockerfile)
+echo ${docker_file}
+if [[ ${docker_file} != "docker/Dockerfile" ]]; then
+  echo "Cannot find docker/Dockerfile!"
+  exit
+fi
+
+# Building the image
+time_tag=$(date +%F-%H-%M-%S)
```

Comment:
Consider making this a more general flag (e.g. TAG_NAME) that defaults to current time.